oracle/price-feeder/oracle/oracle.go

```diff
@@ -497,6 +512,20 @@ func (o *Oracle) checkWhitelist(params oracletypes.Params) {
 	}
 }
 
+// filterPricesByDenomList takes a list of DecCoins and filters out any
+// coins that are not in the provided DenomList.
+func filterPricesByDenomList(dc sdk.DecCoins, dl oracletypes.DenomList) sdk.DecCoins {
```

Comment:
nit, can we rename these from `dc` and `dl` to like `coinPrices` and `denomList` or something more verbose / clear